### PR TITLE
Add new site for store

### PR DESCRIPTION
**Pull request by Hime-Dev**
Add new site for store

### CODE REVIEW

File: websites/A/AnimeWho/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "name": "Prenses",
+    "id": "244508250630782977"
+  },
+  "service": "AnimeWho?",
+  "description": {
+    "tr": "AnimeWho? Türkçe anime & manga izleyip, okuyabileceğiniz bir sitedir.",
+    "en": "AnimeWho? is a site where you can watch Turkish subtitle anime & read manga."
+  },
+  "url": ["www.animewho.com", "animewho.com"],
+  "version": "V4 - Beta",
```
Comment (Bas950) [inline websites/A/AnimeWho/dist/metadata.json:12]: version: 1.0.0

File: websites/A/AnimeWho/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "author": {
+    "name": "Prenses",
+    "id": "244508250630782977"
+  },
+  "service": "AnimeWho?",
```
Comment (Bas950) [inline websites/A/AnimeWho/dist/metadata.json:6]: service name and folder name must be the same (question mark included)

**Review by Bas950 — CHANGES_REQUESTED** [review]
Fix deepscan errors, upload proof of it working

**Comment by Hime-Dev** [comment]
unknown
@Bas950

File: websites/A/AnimeWho/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    "name": "Zamion101",
+    "id": "117331106868625411"
+  }],
+  "service": "AnimeWho",
+  "description": {
+    "tr": "AnimeWho? Türkçe anime & manga izleyip, okuyabileceğiniz bir sitedir.",
+    "en": "AnimeWho? is a site where you can watch Turkish subtitle anime & read manga."
+  },
+  "url": ["www.animewho.com", "animewho.com"],
+  "version": "0.1.0",
```
Comment (Bas950) [inline websites/A/AnimeWho/dist/metadata.json:16]: version: 1.0.0

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/A/AnimeWho/dist/metadata.json (hunk 3 above)
Comment (Hime-Dev) [reply websites/A/AnimeWho/dist/metadata.json]: fixed

**Review by Bas950 — APPROVED** [review]

**Pull request merged**